### PR TITLE
feat: update README.md

### LINKED ISSUE #14: Mention debug/info logs file in README
Maybe add a section around debugging that tells users to look at these files.

### PR DESCRIPTION
**Pull request by arvgord**
- Add info about Healthcare DICOM Editor role
- Mention about logs folder

Closes #14, #15

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,29 +15,39 @@ Before running DICOMFuse, ensure that you've installed the following
...
 
-> [libfuse](https://github.com/libfuse/libfuse) has been supported in Linux
-> kernel since 2.6.14 version.
+> FUSE has been supported in Linux kernel since 2.6.14 version.
 
 ### Credentials
 
 You can access the Cloud Healthcare API using
 [Google application default credentials](https://cloud.google.com/docs/authentication/production#howtheywork).
+Set Healthcare DICOM Editor role for
```
Comment (dzelemba) [inline README.md]: Switch this paragraph to:

DICOMFuse will use [application default credentials](https://cloud.google.com/docs/authentication/production#howtheywork) to authenticate access to the Cloud Healthcare API. The identity must have the permissions within the [Healthcare DICOM Editor role](https://cloud.google.com/healthcare/docs/concepts/access-control#roles). 

Alternatively you can specify a service account key via the `--keyFile` flag.

The easiest way to setup credentials is to use your user credentials with the [gcloud tool](http://cloud/sdk/gcloud/):

```
gcloud auth application-default login
```

File: README.md [hunk 2]
```diff
@@ -15,29 +15,39 @@ Before running DICOMFuse, ensure that you've installed the following
...
 ### Credentials
 
 You can access the Cloud Healthcare API using
 [Google application default credentials](https://cloud.google.com/docs/authentication/production#howtheywork).
+Set Healthcare DICOM Editor role for 
+[service account](https://cloud.google.com/healthcare/docs/how-tos/authentication).
 You can set the `GOOGLE_APPLICATION_CREDENTIALS` environment variable, or you
 can use `--keyFile` option when you launch DICOMFuse.
 
+> More info about [roles](https://cloud.google.com/healthcare/docs/concepts/access-control#roles).
```
Comment (dzelemba) [inline README.md]: Can remove this after applying my above suggestion.

File: README.md (hunk 1 above)
Comment (arvgord) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (arvgord) [reply README.md]: Done

File: README.md [hunk 3]
```diff
@@ -15,29 +15,40 @@ Before running DICOMFuse, ensure that you've installed the following
...
-can use `--keyFile` option when you launch DICOMFuse.
+DICOMFuse will use 
+[application default credentials](https://cloud.google.com/docs/authentication/production)
+to authenticate access to the Cloud Healthcare API. The identity must have the 
+permissions within the 
+[Healthcare DICOM Editor role](https://cloud.google.com/healthcare/docs/concepts/access-control#roles).  
+Alternatively, you can specify a service account key via the `--keyFile` flag.
+The easiest way to set up credentials is to use your user credentials with the 
+[gcloud tool](https://cloud.google.com/sdk/gcloud/):  
+`gcloud auth application-default login`
```
Comment (dzelemba) [inline README.md]: Add a line of whitespace before the `gcloud auth ...` command.

Comment (dzelemba) [reply README.md]: Nevermind looks I can make an edit myself :)

**Pull request merged**